tf2_filter/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>tf2_filter</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="ingo.luetkebohle@de.bosch.com">Luetkebohle Ingo (CR/AEX3)</maintainer>
+  <license>APL</license>
```

Comment:
Propose to use 'Apache-2.0' from https://spdx.org/licenses/

Reply:
+1

Reply:
Done.